src/components/items/selected_item.js

```diff
@@ -3,16 +3,22 @@ import PropTypes from "prop-types";
 import CloseIcon from "react-icons/lib/md/close";
 import IconButton from "@material-ui/core/IconButton";
 import ItemLabel from "./item_label";
-
+import classnames from "classnames";
 import styles from "./selected_item.scss";
+import itemStyles from "./item.scss";
 
-const SelectedItem = ({ item, height, group }) => (
+const SelectedItem = ({ item, height, group, isLocked }) => (
   <div
-    className={group ? styles.with_grouping : styles.selected_item}
+    className={classnames({
+      [styles.with_grouping]: group,
+      [styles.selected_item]: !group,
+      [itemStyles.disabled]: isLocked(item),
+      [itemStyles.selected]: isLocked(item)
```

Comment:
are you sure we need this?.... it's selected item component, it is only for selected